Airports/EGKK/Sids.txt

```diff
@@ -54,6 +49,9 @@ SID:EGKK:26L:#BIG8M:D258B DET31 DET29 NEXOX D123R BIG
 SID:EGKK:26R:#BIG8V:D260C DET31 DET29 NEXOX D123R BIG
```

Comment:
Think these Removed XXXX/XX can be binned as well. @hsugden ?

Reply:
Yes actually, as we'll be establishing a new procedure for EGKK -> EGLL repo flights